AK/StringBuilder.cpp

```diff
@@ -85,19 +85,19 @@ void StringBuilder::appendf(const char* fmt, ...)
     va_end(ap);
 }
 
-ByteBuffer StringBuilder::to_byte_buffer()
+ByteBuffer StringBuilder::to_byte_buffer() const
 {
-    m_buffer.trim(m_length);
-    m_length = 0;
-    return move(m_buffer);
+    ByteBuffer buffer { m_buffer };
+    buffer.trim(m_length);
+    return buffer;
```

Comment:
This will (quite deceptively) still create a `ByteBuffer` that refers to the same internal `ByteBufferImpl` as `m_buffer`.
Instead, we can just do `return m_buffer.isolated_copy();`

Reply:
Sorry, I supposed it's copy-on-write.